ocaml/libs/tgroup/tgroup.mli

```diff
@@ -46,12 +46,23 @@ module Cgroup : sig
   type t = string
 
   val dir_of : Group.t -> t
-  (** [dir_of group] returns the full path of the cgroup directory corresponding to the group [group].*)
+  (** [dir_of group] returns the full path of the cgroup directory corresponding
+  to the group [group].*)
 
   val init : unit -> unit
-  (** [init ()] initializes the hierachy of cgroups associated to all [Group.t] types.
+  (** [init ()] initializes the hierachy of cgroups associated to all [Group.t]
+  types.
   *)
 
   val set_cgroup : Group.Originator.t -> unit
-  (** [set_cgroup o] sets the current xapi thread in a cgroup based on the originator [o].*)
+  (** [set_cgroup o] sets the current xapi thread in a cgroup based on the
+  originator [o].*)
```

Comment:
seems like this reformatting should be in the first commit

Reply:
I modified that, and some spelling and grammar.